apollo-core/src/main/java/com/ctrip/framework/apollo/core/utils/StringUtils.java

```diff
@@ -355,11 +355,11 @@ public String format(T obj) {
 
   public static <T> String join(Collection<T> collection, String separator,
```

Comment:
Seems this method is only used in tests, let's just remove it

Reply:
Removed method and changed the commit message to 'delete unused method StringUtils.join(..)'